src/docs/product/integrations/integration-platform/ui-components/formfield.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: FormField Components
+sidebar_order: 4
+description: "Learn about FormField components."
+---
+
+The [issue linking](/product/integrations/integration-platform/ui-components/issue-link) and [alert rule action](/product/integrations/integration-platform/ui-components/alert-rule-action) UI components can use `FormField` components to display attributes in their respective modals. These components can render custom forms with inputs specified according to the schemes below. They behave similarly to their HTML equivalents.
+
+## Text
+
+### Schema
+
+```json {filename:schema.json}
+{
+  "type": "text",
+  "label": <String>,
+  "name": <String>,
+  (optional) default: <String>,
+}
+```
+
+### Attributes
+
+- `label` - (Required) Label text to be rendered for the form field
+- `name` - (Required) Value to use in `name` attribute of the field
+- `default` - default to pre-populate with. Options include `issue.title` and `issue.description`. Only used for [issue link](/product/integrations/integration-platform/ui-components/issue-link) components.
+  - `issue.title` - title of the Sentry issue
+  - `issue.description` - description of the Sentry issue
+
+## Textarea
+
+### Schema
+
+```json {filename:schema.json}
+{
+  "type": "textarea",
+  "label": <String>,
+  "name": <String>,
+  (optional) default: <String>,
+}
+```
+
+### Attributes
+
+- `label` - (Required) Label text to be rendered for the form field
+- `name` - (Required) Value to use in `name` attribute of the field
+- `default` - default to pre-populate with options include `issue.title` and `issue.description`. Only used for [issue link](/product/integrations/integration-platform/ui-components/issue-link) components.
+  - `issue.title` - title of the Sentry issue
+  - `issue.description` - description of the Sentry issue
+
+## Select
+
+### Schema
+
+```json {filename:schema.json}
+{
+  "type": "select",
+  "label": <String>,
+  "name": <String>,
+  "uri": <URI>,
+  "async": <Bool>,
+  "options": <Array<String, String>>,
+  "depends_on": <Array<String>> (optional)
+  "skip_load_on_open": <Array<String>> (optional)
```

Comment:
Can we remove `(optional)` from the codeblocks? It's not consistent with the text below, and it looks more consistent without em.